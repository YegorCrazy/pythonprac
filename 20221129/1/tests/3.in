class C(metaclass=dump):
	def perf(self, a=5, b=6, d=8):
		return a * b * d

c = C()
print(c.perf(6, d=10))
print(c.perf(6))
