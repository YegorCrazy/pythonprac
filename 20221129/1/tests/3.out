perf: (6,), {'d': 10}
360
perf: (6,), {}
288
